### DISCUSSION TITLE
Moving the methods site to a more extensible framework for future expansion

### DISCUSSION THREAD

**Discussion by pglevy** [Bixal/methods#99] [Ideas]
Some issues to consider:
- There may be other collections of content that it will make sense to incorporate into the existing site: a "recruiting" playbook, visual and branding guidelines, things relating to service design, etc.
- If we're comfortable going all-in on a digital format, we don't have to be as constrained by the "card" format, which was intended to work for printing as well. While this places some helpful constraints on the section layout and amount of content, we could retain some of these as general guidance while allowing for more flexibility with both page structure and overall site structure.
- The current site is probably unnecessarily complex for what we need. Moving to something like [Just the Docs](https://pmarsceill.github.io/just-the-docs/) provides a nice framework for documentation while simplifying the file and code structure we would have to maintain.

@Bixal/methods

**Comment by pglevy** [comment]
Here's a quick [proof-of-concept using Just the Docs](https://pglevy.github.io/jtd-demo/).

**Reply by sofya-UX** [reply]
This looks super simple and clear, and they have their UI components @katharine-carr :)

**Reply by pglevy** [reply]
The built-in search is pretty nice too.

**Comment by katharine-carr** [comment]
Not bad! I like the clean design. Will help us get around weird branding issues too.

**Reply by sofya-UX** [reply]
The biggest sell for a "tech-unfriendly" individual like me is that it looks really simple to get behind the scenes to maintain and edit the copy [if we have to]. Is this so @pglevy ?

**Reply by pglevy** [reply]
There's a little bit of ramp-up, but it's not too complicated. Overall, it makes it really easy to structure the content.

**Reply by katharine-carr** [reply]
And YAY it has search! Woot!
